drivers/bridge/setup_ip_tables.go

```diff
@@ -11,20 +11,21 @@ import (
 
 // DockerChain: DOCKER iptable chain name
 const (
-	DockerChain = "DOCKER"
+	DockerChain    = "DOCKER"
+	IsolationChain = "ISOLATION"
```

Comment:
Can we name it `DOCKER-ISOLATION`? Just to follow a convention, we are already calling overlay chain as `DOCKER-OVERLAY` so people looking at it will know that this is for docker